tasks/subscription-manager.yml

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Call subscription-manager
+  redhat_subscription:
+    state: "{{ rhc_state | d('present') }}"
+    username: "{{ rhc_auth.login.username | d(omit) }}"
+    password: "{{ rhc_auth.login.password | d(omit) }}"
+    activationkey: "{{ rhc_auth.activation_keys.keys | join(',')
+  if ( rhc_auth.activation_keys.keys is defined ) else omit }}"
```

Comment:
```suggestion
      if ( rhc_auth.activation_keys.keys is defined ) else omit }}"
```
suggest to indent

Reply:
ah ok, will do, thanks (... i shamelessly copied it from other roles :) )